wrlinux8/templates/Makefile

```diff
@@ -0,0 +1,15 @@
+export SHARED_DIR=../../shared/templates/
+export PYTHONPATH=../../shared
+OUTPUT:=output
+OUTPUT_BASH:=$(OUTPUT)/bash
+CSV_DIR:=csv
+
```

Comment:
I wonder what is the Makefile good for. We don't have makefiles anywehere else than in WRLinux directories. We use CMake for a long time.

Reply:
I think the Makefiles can be removed, I can't find any place where they're called.

Reply:
Thanks for the reminding. I will remove the Makefile from WRLinux directories.